evaluation/swe_bench/run_infer.py

```diff
@@ -86,21 +87,34 @@ def get_instruction(instance: pd.Series, metadata: EvalMetadata):
     return instruction
 
 
+# typically the namespace
+DOCKER_IMAGE_PREFIX = os.environ.get('EVAL_DOCKER_IMAGE_PREFIX', 'docker.io/xingyaoww/')
```

Comment:
I'd rather not hard-code your docker here, if we can help it :)

Reply:
Basically, all the SWE-Bench training docker is uploaded to my personal DockerHub account (bc there are too many of those!), and I think this can serve as a convenient default.

Previously, we asked ppl to manually download it for inference: https://github.com/All-Hands-AI/OpenHands/tree/main/evaluation/swe_bench#download-docker-images. But this is less of an idea because we want run_infer.sh to just work out-of-the-box, hence this change.

If we don't hard-code it, we probably need to tell people to set this explict as env var in the README.md, which probably will add more inconvenience than helping?

Reply:
We should at least move it to an openhands or allhands repo though. We can make this a TODO

Reply:
added a todo!